website/templates/emails/withdrawal_request_declined.html.mako

```diff
@@ -6,9 +6,25 @@
     <%!
         from website import settings
     %>
-        Dear ${requester.fullname},<br>
-        <br>
-        Your request to withdraw your ${document_type} <a href="${reviewable.absolute_url}">"${reviewable.title}"</a> from ${reviewable.provider.name} has been declined by the service moderators. Login and visit your ${document_type} to view their feedback. The ${document_type} is still publicly available on ${reviewable.provider.name}.
+        % if document_type == 'registration':
+            Your request to withdraw your registration <a href="${reviewable.absolute_url}">"${reviewable.title}"</a> from ${reviewable.provider.name} has been declined by the service moderators. The registration is still publically available on ${reviewable.provider.name}.
```

Comment:
Apparently 'publicly' is the primary form of the word, and was what was in the doc.

```suggestion
            Your request to withdraw your registration <a href="${reviewable.absolute_url}">"${reviewable.title}"</a> from ${reviewable.provider.name} has been declined by the service moderators. The registration is still publicly available on ${reviewable.provider.name}.
```